components/x-live-blog-post/src/LiveBlogPost.jsx

```diff
@@ -27,7 +27,7 @@ const LiveBlogPost = (props) => {
 			</div>
 			{isBreakingNews && <div className={styles['live-blog-post__breaking-news']}>Breaking news</div>}
 			{title && <h1 className={styles['live-blog-post__title']}>{title}</h1>}
-			<div className={styles['live-blog-post__body']} dangerouslySetInnerHTML={{ __html: bodyHTML || content }} />
+			<div className={`${styles['live-blog-post__body']} n-content-body`} dangerouslySetInnerHTML={{ __html: bodyHTML || content }} />
```

Comment:
you'll need to update the snapshots 🙂

Reply:
@apaleslimghost I have updated the snapshots but that made the tests fail, so I have reverted that commit.